osu.Game/Input/Bindings/GlobalActionContainer.cs

```diff
@@ -378,5 +379,9 @@ public enum GlobalAction
 
         [LocalisableDescription(typeof(GlobalActionKeyBindingStrings), nameof(GlobalActionKeyBindingStrings.ToggleReplaySettings))]
         ToggleReplaySettings,
+
+        // Editor (cont)
```

Comment:
Wrong comment and is probably not needed. This enum will only keep growing and be disorganized. I would remove all the other comments instead.

Reply:
tbh, I think this probably makes sense. 

I spend the past couple days trying to re-organize the enum and then fix the bindings during the next ream migration, but that really wasn't working. I'm sure it could be done, but probably not by me